test/e2e/main_test.go

```diff
@@ -26,19 +25,18 @@ import (
 	"github.com/pkg/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/tools/clientcmd"
 )
 
 var f *framework.Framework
 
 func TestMain(m *testing.M) {
-	usr, _ := user.Current()
-	kubeConfigPath := flag.String("kubeconfig", usr.HomeDir+"/.kube/config", "kube config path, default: $HOME/.kube/config")
+	kubeConfigPath := flag.String("kubeconfig", clientcmd.RecommendedHomeFile, "kube config path, default: $HOME/.kube/config")
 	opImageName := flag.String("operator-image", "", "operator image, e.g. quay.io/coreos/cluster-monitoring-operator")
 
 	flag.Parse()
 
-	var err error
-	f, err = framework.New(*kubeConfigPath, *opImageName)
+	f, err := framework.New(*kubeConfigPath, *opImageName)
```

Comment:
@s-urbaniak this variable declaration is shadowing the f declared on line 31. Not sure if this breaks the program but it is a little suspicious at the least.

Reply:
ah, good catch :+1: yes this shadowing should break.

Reply:
@squat changed, PTAL